### PR TITLE
Allow skip on module level messages

### PR DESCRIPTION
**Pull request by snudler6**
Thanks for submitting a PR!

Please make sure to check for the following items:
- [ ] Add unit tests and integration tests where applicable.  
      If you've added an error code or changed an error code behavior,
      you should probably add or change a test case file under `tests/test_cases/` and add 
      it to the list under `tests/test_definitions.py`.  
      If you've added or changed a command line option,
      you should probably add or change a test in `tests/test_integration.py`.
- [ ] Add a line to the release notes (docs/release_notes.rst) under "Current Development Version".  
      Make sure to include the PR number after you open and get one.
   
Please don't get discouraged as it may take a while to get a review.

### CODE REVIEW

**Comment by snudler6** [comment]
resolves #418

File: docs/release_notes.rst [hunk 1]
```diff
@@ -18,6 +18,7 @@ New Features
 * Extend support for detecting missing argument description in Numpy style
   docstrings (#407).
 * Added support for Python 3.8 (#423).
+* Allow skipping on errors on module level docstring (#427).
```
Comment (Nurdok) [inline docs/release_notes.rst]: Mention the feature name (noqa), e.g.:

Allow skipping errors on module level docstring via `#noqa` (#427).

Comment (snudler6) [reply docs/release_notes.rst]: Done.

**Comment by bsamseth** [comment]
While this works for the test case, it doesn't work if there is a comment before the module docstring. An example that still reports and error:

```python
# -*- coding: utf-8 -*-
# noqa: D400
"""No period at the end""" 
```
Encoding pragmas, or `#!/path/to/python` comments are quite common, so I think this should be fixed to account for this.

**Pull request merged**